package.json

```diff
@@ -67,6 +67,7 @@
     "@storybook/addon-storysource": "^6",
     "@storybook/addon-viewport": "^6",
     "@storybook/react": "^6",
+    "@testing-library/jest-dom": "^5.11.9",
```

Comment:
I added this optional testing-library module for easier appearance-disappearance testing. See official [docs](https://testing-library.com/docs/guide-disappearance/). Other methods didn't work in some cases (I still got loading spinner when querying with findBy).

Reply:
The `waitForElementToBeRemoved()` is not from `jest-dom`. You can see the import is from the React testing library.

Nonetheless, it's a useful library which includes some [useful DOM testing helpers](https://github.com/testing-library/jest-dom#jest-dom).

For example, instead of testing `expect(element).toBeTruthy()`, you can test for [`expect(element).toBeInTheDocument()`](https://github.com/testing-library/jest-dom#tobeinthedocument)